cgi-bin/Apache/LiveJournal.pm

```diff
@@ -198,6 +198,47 @@ sub totally_down_content
     return OK;
 }
 
+sub send_concat_res_response {
+    my $apache_r = $_[0];
+    my $args = $apache_r->args;
+    my $uri = $apache_r->uri;
+
+    my $dir = ( $LJ::STATDOCS // $LJ::HTDOCS ) . $uri;
+    return 404
```

Comment:
If you look around line 370 of the current version of the module, you can see what we do to print our 404 page, but that assumes the request status is already 404.